public/js/dashboard.js

```diff
@@ -51,3 +52,11 @@ if (document.querySelector(".email-card")) {
     option.addEventListener("click", sendCommunicationOption);
   });
 }
+
+if (document.querySelector(".remove-fxm")) {
+  const removeMonitorButton = document.querySelector(".remove-fxm");
```

Comment:
minor nit, feel free to ignore... but if we move the `const` above the `if`, we can just do:

```js
if (removeMonitorButton) {...}
```

and avoid a second DOM traversal.

Reply:
also nice.